src/test/java/eu/dissco/dataexporter/utils/TestUtils.java

```diff
@@ -56,35 +58,34 @@ public static ExportJobRequest givenJobRequest() {
         .withType("export-job")
         .withAttributes(new Attributes()
             .withExportType(ExportType.DOI_LIST)
-            .withParams(givenParams())));
-
-  }
-
-  public static Params givenParams() {
-    return new Params().withAdditionalProperty("$['ods:organisationID']",
-        "https://ror.org/0566bfb96");
+            .withSearchParams(givenSearchParams())
+            .withTargetType(Attributes.TargetType.HTTPS_DOI_ORG_21_T_11148_894_B_1_E_6_CAD_57_E_921764_E)));
```

Comment:
This type looks strange, shouldn't that be DigitalSpecimen as the enum is?

Reply:
So we have the `JobRequest` object which is generated by jsonschema2pojo, then we have the `ExportJob` which is an immutable object we build based on the request. 

TargetType is from the `JobRequest` object and is defined by the following jsonschema:
```
  "targetType": {
              "description": "FDO type of the objects whose data is being exported",
              "type": "string",
              "enum": [
                "https://doi.org/21.T11148/894b1e6cad57e921764e",
                "https://doi.org/21.T11148/bbad8c4e101e8af01115"
              ]
            }
```

So it annoyingly pulls the fdo type url as the enum name :upside_down_face:  it's only explicitly used in testing, though, so i'm okay keeping it as is.